<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>pqHistogramDialog</class>
 <widget class="QDialog" name="pqHistogramDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>952</width>
    <height>434</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="Confirmation">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>390</y>
     <width>176</width>
     <height>27</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>1</x>
     <y>1</y>
     <width>952</width>
     <height>371</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="pqHistogramWidget" name="HistogramWidget" native="true">
      <property name="geometry">
       <rect>
          <x>0</x>
          <y>0</y>
          <width>952</width>
          <height>300</height>
         </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>950</width>
        <height>300</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="pageEnableLog">
       <widget class="QPushButton" name="UseLogScale">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>110</width>
          <height>27</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Use Log Scale</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="pageDisableLog">
       <widget class="QPushButton" name="DisableLogScale">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>10</y>
          <width>131</width>
          <height>27</height>
         </rect>
        </property>
        <property name="text">
         <string>Disable Log Scale</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>pqHistogramWidget</class>
   <extends>QWidget</extends>
   <header>pqHistogramWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
